dex/networks/eth/contracts/v0/BinRuntimeV0.go

```diff
@@ -3,4 +3,4 @@
160a01b900460ff1660038111156104a3576104a3610a04565b1480156104c957506000838152602081905260409020600501546001600160a01b031633145b80156105425750826002836040516020016104e691815260200190565b60408051601f1981840301815290829052610500916108cf565b602060405180830381855afa15801561051d573d6000803e3d6000fd5b5050506040513d601f19601f820116820180604052508101906105409190610872565b145b9392505050565b600081815260208190526040812060016005820154600160a01b900460ff16600381111561057957610579610a04565b148015610592575060048101546001600160a01b031633145b801561054257506003015442101592915050565b3233146105b257600080fd5b6000805b828110156106f657368484838181106105d1576105d1610a1a565b6020604091820293909301838101356000908152938490529220919250600190506005820154600160a01b900460ff16600381111561061257610612610a04565b1461061c57600080fd5b60058101546001600160a01b0316331461063557600080fd5b81602001356002836000013560405160200161065391815260200190565b60408051601f198184030181529082905261066d916108cf565b602060405180830381855afa15801561068a573d6000803e3d6000fd5b5050506040513d601f19601f820116820180604052508101906106ad9190610872565b146106b757600080fd5b60058101805460ff60a01b1916600160a11b1790558135815560018101546106df90856109bb565b9350505080806106ee906109d3565b9150506105b6565b50604051600090339083908381818185875af1925050503d8060008114610739576040519150601f19603f3d011682016040523d82523d6000602084013e61073e565b606091505b509091505060018115151461075257600080fd5b50505050565b60006020828403121561076a57600080fd5b81356001600160a01b038116811461054257600080fd5b6000806020838503121561079457600080fd5b823567ffffffffffffffff808211156107ac57600080fd5b818501915085601f8301126107c057600080fd5b8135818111156107cf57600080fd5b8660208260071b85010111156107e457600080fd5b60209290920196919550909350505050565b6000806020838503121561080957600080fd5b823567ffffffffffffffff8082111561082157600080fd5b818501915085601f83011261083557600080fd5b81358181111561084457600080fd5b8660208260061b85010111156107e457600080fd5b60006020828403121561086b57600080fd5b5035919050565b60006020828403121561088457600080fd5b5051919050565b6000806040838503121561089e57600080fd5b50508035926020909101359150565b600481106108cb57634e487b7160e01b600052602160045260246000fd5b9052565b6000825160005b818110156108f057602081860181015185830152016108d6565b818111156108ff576000828501525b509190910192915050565b8781526020810187905260408101869052606081018590526001600160a01b038481166080830152831660a082015260e0810161094a60c08301846108ad565b98975050505050505050565b600060e08201905082518252602083015160208301526040830151604083015260608301516060830152608083015160018060a01b0380821660808501528060a08601511660a0850152505060c08301516109b460c08401826108ad565b5092915050565b600082198211156109ce576109ce6109ee565b500190565b60006000198214156109e7576109e76109ee565b5060010190565b634e487b7160e01b600052601160045260246000fd5b634e487b7160e01b600052602160045260246000fd5b634e487b7160e01b600052603260045260246000fdfea2646970667358221220a36f1109c6419ae7a881c27974458b5cf52dbb692ee3f12b0e0f81ca0e4dde6d64736f6c63430008060033"
```

Comment:
This is not about the pr.

I thought that changing this value would change the tx that makes the contract and thereby change the contract address, but it does not. I wonder what decides contract addresses.